src/stavehairpin.ts

```diff
@@ -132,14 +132,7 @@ export class StaveHairpin extends Element {
   }
 
   setRenderOptions(options: StaveHairpinRenderOptions): this {
-    if (
-      options.height != null &&
-      options.yShift != null &&
-      options.leftShiftPx != null &&
-      options.rightShiftPx != null
-    ) {
-      this.renderOptions = options;
-    }
+    this.renderOptions = options;
```

Comment:
Just checking on the reasoning behind this deletion.

So now it's OK to `null` out certain options?

Reply:
According to the type definition, these elements cannot be null.